vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ModuleOverviewManager.java

```diff
@@ -0,0 +1,84 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.model.IChoice;
+import edu.asu.diging.vspace.core.model.IModule;
+import edu.asu.diging.vspace.core.model.ISequence;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.BranchingPoint;
+import edu.asu.diging.vspace.core.services.IModuleManager;
+import edu.asu.diging.vspace.core.services.IModuleOverviewManager;
+import edu.asu.diging.vspace.core.services.impl.model.ModuleOverview;
+import edu.asu.diging.vspace.core.services.impl.model.SequenceOverview;
+import edu.asu.diging.vspace.core.services.impl.model.SlideOverview;
+
+@Service
+public class ModuleOverviewManager implements IModuleOverviewManager {
+    
+    @Autowired
+    private IModuleManager moduleManager;
+    
+    public ModuleOverview getModuleOverview(String moduleId) {
+        IModule module = moduleManager.getModule(moduleId);
+        ISequence startSequence = module.getStartSequence();
+        
+        List<ISequence> sequences = moduleManager.getModuleSequences(moduleId);
+        
+        SequenceOverview startSequenceNode = createSequenceOverviewNode(startSequence);
```

Comment:
this is of type sequenceOverview, why is the variable called `startSequenceNode`?